dataeng/jobs/analytics/TableauConfigBackup.groovy

```diff
@@ -0,0 +1,52 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+class TableauConfigBackup{
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("tableau-config-backup"){
```

Comment:
All new jobs should have a description. Can you add one please?

Reply:
Done that, thanks for pointing